backend/app/apps.py

```diff
@@ -24,13 +29,17 @@ def load_appstream():
 
             search_keywords = apps[appid].get("keywords")
 
+            project_license = apps[appid].get("project_license", "")
+
             # order of the dict is important for attritbute ranking
             search_apps.append(
                 {
                     "id": utils.get_clean_app_id(appid),
                     "name": apps[appid]["name"],
                     "summary": apps[appid]["summary"],
                     "keywords": search_keywords,
+                    "project_license": project_license,
+                    "is_free_license": AppStream.license_is_free_license(project_license),
```

Comment:
I don't like how we have two terms `free license` and `free software` now. Would like them to be aligned.

Reply:
I agree, I think on the search result page the term "free software" makes most sense, because you're currently trying to find software, and "Show only software with a free license" is a mouthful. I will change this field to `is_free_software`.